exonum/src/runtime/migrations.rs

```diff
@@ -100,12 +225,14 @@ pub trait MigrateData {
     /// be executed successively in the specified order, flushing the migration to the DB
     /// after each script is completed.
     ///
-    /// Scripts are not expected to fail; the failure should be eagerly signalled via an error.
+    /// Inability to migrate data should be eagerly signalled via an error. While a script may
```

Comment:
This is a bit unclear. I think we should expand the explanations with separate points:
1. How one should report a migration error and what will be the core reaction;
2. What's the difference between eager and lazy error reporting and why the first one is preferred;
3. How one should react on a migration abortion and what will be the core reaction.